src/state/hooks.ts

```diff
@@ -0,0 +1,5 @@
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux'
+import { AppDispatch, AppState } from 'state'
+
+export const useAppDispatch = () => useDispatch<AppDispatch>()
```

Comment:
you can't do the same thing on this line as the next line?

Reply:
following this, https://react-redux.js.org/using-react-redux/usage-with-typescript

i'd rather keep it in line with the docs